schemas/api/intentMetadata.schema.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "$id": "https://fdc3.finos.org/schemas/next/intentMetadata.schema.json",
+    "$id": "https://fdc3.finos.org/schemas/next/api/intentMetadata.schema.json",
     "title": "IntentMetadata",
     "description": "The interface used to describe an intent within the platform.",
     "type": "object",
```

Comment:
In this type, please remove `name` from the required array, due to https://github.com/finos/FDC3/pull/926 which should get merged today